ui/src/elements/ngm-dashboard.ts

```diff
@@ -333,10 +336,10 @@ export class NgmDashboard extends LitElementI18n {
 
     apiClient.duplicateProject(createProject).then(r => r.json())
       .then(id => {
-        apiClient.getProject(id).then(r => r.json()).then(project => {
-          this.selectedTopicOrProject = project;
+        apiClient.getProject(id).then(r => r.json()).then(async project => {
```

Comment:
For consistency I suggest to use promise here.

Reply:
@gberaudo not sure what you mean by using promise. I made it consistent with line 112.